woocommerce/woocommerce-ios
# [In-app Purchases] Add logging to InAppPurchaseStore

Closes: #7934 


### Description


Adds logging for all important events in the IAP store.

### Testing instructions


Go to Hub menu > IAP Debug. As the product list loads, notice the messages in the console with the `[💰IAP Store]` prefix.

### Screenshots


![Screen Shot 2022-10-25 at 16 28 07](https://user-images.githubusercontent.com/8739/197801361-e7ed27e5-d893-47c6-8d0c-c1a486091d97.png)

---
